omod-2.3/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs2_3/OrderSearchHandler2_3.java

```diff
@@ -90,11 +92,12 @@
 	            REQUEST_PARAM_CANCELED_OR_AUTO_EXPIRE_ON_OR_BEFORE_DATE,
 	            REQUEST_PARAM_ACTION,
 	            REQUEST_PARAM_FULFILLER_STATUS,
+	            REQUEST_PARAM_INCLUDE_NULL_FULFILLER_STATUS,
 	            REQUEST_PARAM_EXCLUDE_CANCELED_AND_EXPIRED,
 	            REQUEST_PARAM_INCLUDE_VOIDED).build();
 	
 	private final SearchConfig searchConfig = new SearchConfig("default", RestConstants.VERSION_1
-	        + "/order", Arrays.asList("2.4.*"), searchQuery);
+	        + "/order", Arrays.asList("2.3.*"), searchQuery);
```

Comment:
let's add 2.4.* back in here as well for future releases (I'll add that quickly in a post-commit, want to get this through and up on our build as soon as possible...)

Reply:
Thanks @mogoodrich !